packages/web-components/src/components/va-accordion/va-accordion.tsx

```diff
@@ -96,8 +100,42 @@ export class VaAccordion {
     if (!this.isScrolledIntoView(clickedItem)) {
       clickedItem.scrollIntoView();
     }
+
+    // Check if all accordions are open or closed due to user clicks
+    this.accordionsOpened();
   }
 
+  private accordionsOpened() {
```

Comment:
Could you please add a test for this?

Reply:
@k80bowman an additional test has been added to cover this